configs/mock/rollup_config.toml

```diff
@@ -5,7 +5,7 @@ prover_da_pub_key = ""
 
 [da]
 sender_address = "0000000000000000000000000000000000000000000000000000000000000000"
-db_path = "da-db"
+db_path = "test-da-dbs"
```

Comment:
Please revert changes in those config files.

`test-da-dbs` is removed in a different PR and this value is set to `resources/dbs/da-db`

Reply:
Done